src/views/includes/subscribe-view.jade

```diff
@@ -36,4 +36,4 @@
     .subscribe__thank-you-wrapper
       h4.headline.headline--m= subscribeView['thanks-title']
       p.subscribe__thank-you-msg!= subscribeView['thanks-msg'][0]
-      p.subscribe__thank-you-msg!= subscribeView['thanks-msg'][1]
+      //p.subscribe__thank-you-msg!= subscribeView['thanks-msg'][1]
```

Comment:
@odincov  Got it! I think we should bring it back, once donations are implemented.